packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx

```diff
@@ -143,6 +150,25 @@ const NftOrderStatus: React.FC<Props> = (props) => {
           <SpinningLoader height='14px' width='14px' borderWidth='3px' />
         </Wrapper>
       )}
+      {props.orderFlow.status === NftOrderStatusEnum.READY_FOR_SALE && (
```

Comment:
small thing but READY_FOR_SALE sounds like it is ready for sale not loading.

Reply:
yeah, should be POST_LISTING actually, thanks